generation/3d_ldm/3d_ldm_tutorial.ipynb

```diff
@@ -0,0 +1,1206 @@
+{
```

Comment:
<p>Rather than referring to it as the 'MONAI Generative Models package,' let's update it to 'MONAI Generative components.' Alternatively, we could mention that the MONAI Generative Models package is already integrated into the MONAI core. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>